src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderAssessmentRepository.java

```diff
@@ -10,21 +10,12 @@
 public interface OffenderAssessmentRepository extends CrudRepository<OffenderAssessment, OffenderAssessment.Pk> {
     Optional<OffenderAssessment> findByBookingIdAndAssessmentSeq(Long bookingId, Integer assessmentSeq);
 
-    @Query("""
```

Comment:
Removed as it is a duplicate of the offenderNos version (except for being the singular)